tools/clang/test/CodeGenHLSL/optForNoOpt3.hlsl

```diff
@@ -1,17 +0,0 @@
-// RUN: %dxc -E main -T ps_6_0 -Od %s | FileCheck %s
```

Comment:
These files were redundant. Where they were used, the more appropriate location was in dxilvalidation/ where they were copied with the mass test file migration probably with the intent to make this exact change. Rather than alter these as they should be, I resolved what seemed like a mistake.